.NET/Microsoft.Recognizers.Text.DateTime/Japanese/Extractors/JapaneseTimeExtractorConfiguration.cs

```diff
@@ -6,14 +6,56 @@
 
 namespace Microsoft.Recognizers.Text.DateTime.Japanese
 {
-    public class JapaneseTimeExtractorConfiguration : JapaneseBaseDateTimeExtractorConfiguration<TimeType>
+    public class JapaneseTimeExtractorConfiguration : BaseDateTimeOptionsConfiguration, ICJKTimeExtractorConfiguration
     {
-        // e.g: 早上九点
+        public static readonly string HourNumRegex = DateTimeDefinitions.TimeHourNumRegex;
+
+        public static readonly string MinuteNumRegex = DateTimeDefinitions.TimeMinuteNumRegex;
+
+        public static readonly string SecondNumRegex = DateTimeDefinitions.TimeSecondNumRegex;
+
+        public static readonly string HourCJKRegex = DateTimeDefinitions.TimeHourCJKRegex;
+
+        public static readonly string MinuteCJKRegex = DateTimeDefinitions.TimeMinuteCJKRegex;
+
+        public static readonly string SecondCJKRegex = DateTimeDefinitions.TimeSecondCJKRegex;
+
+        public static readonly string ClockDescRegex = DateTimeDefinitions.TimeClockDescRegex;
+
+        public static readonly string MinuteDescRegex = DateTimeDefinitions.TimeMinuteDescRegex;
+
+        public static readonly string SecondDescRegex = DateTimeDefinitions.TimeSecondDescRegex;
+
+        public static readonly string BanHourPrefixRegex = DateTimeDefinitions.TimeBanHourPrefixRegex;
+
+        // e.g: 12時
+        public static readonly string HourRegex = DateTimeDefinitions.TimeHourRegex;
+
+        public static readonly string MinuteRegex = DateTimeDefinitions.TimeMinuteRegex;
+
+        public static readonly string SecondRegex = DateTimeDefinitions.TimeSecondRegex;
+
+        public static readonly string HalfRegex = DateTimeDefinitions.TimeHalfRegex;
+
+        public static readonly string QuarterRegex = DateTimeDefinitions.TimeQuarterRegex;
+
+        // e.g: 十二五十から八|半分|瞬間

```

Comment:
Don't forget to update the comments/examples.

Reply:
yes, I updated them already in both Korean and Japanese config files.